repository.rosinstall

```diff
@@ -29,4 +29,4 @@
 - git:
     local-name: ros_ethercat
     uri: https://github.com/shadow-robot/ros_ethercat.git
-    version: indigo-devel
```

Comment:
This is OK, but please try to remove all the other source dependencies from this file.
Hopefully they all work from debs in kinetic!